modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-binding-workloads-that-are-not-compliant-with-PodSpec_{context}"]
+= Binding secondary workloads that are not compliant with PodSpec
+
+A typical scenario in service binding involves backing service, workload (Deployment), and {servicebinding-title}. Consider a scenario that involves a secondary workload, which can also be an application Operator, not compliant with PodSpec and is between the primary workload (Deployment) and {servicebinding-title}.
+
+For such secondary workload resources, the location of the container path is arbitrary. For service binding, if the secondary workload in a CR is not compliant with the PodSpec, you must specify the location of the container path. This helps in projecting the binding data into the container path specified in the secondary workload of the `ServiceBinding` CR, for example, when you do not want the binding data inside a pod.
+
+{servicebinding-title} provides an option to configure the value of where the container path or secret path is and bind these paths at a custom location. This option is available only for the `binding.operators.coreos.com` API group when the binding data is projected as environment variables.
+
+=== Configuring the custom location of the container path
+Consider a secondary workload CR, which is not compliant with the PodSpec, with containers at the `spec.containers` path:
+
+.Example: Secondary workload CR
+[source,yaml]
+----
+apiVersion: "operator.sbo.com/v1"
+kind: SecondaryWorkload
+metadata:
+    name: secondary-workload
+spec:
+    containers:
+    - name: hello-world
+      image: quay.io/baijum/secondary-workload:latest
+      ports:
+      - containerPort: 9966
+----
+
+The following example shows how to configure the value of where the `spec.containers` path is by specifying it in the `ServiceBinding` CR and bind this path at a `spec.application.bindingPath.containersPath` custom location:
```

Comment:
```suggestion
The following example shows how to configure the `spec.containers` path by specifying a value in the `ServiceBinding` CR and to bind this path to a `spec.application.bindingPath.containersPath` custom location:
```
The syntax here is a bit wordy and confusing. The suggested edit (I hope) clarifies the meaning, though I hope it doesn't alter that meaning.

Reply:
Absolutely, This looks so much better!! Thanks, @bburt-rh :)